docs/source/docsIndex.js

```diff
@@ -0,0 +1,95 @@
+docs['docs/guide.md'] = { guide: true, schema: true, title: 'Schemas', acquit: true, markdown: true };
+docs['docs/schematypes.md'] = { guide: true, schema: true, title: 'SchemaTypes', markdown: true };
+docs['docs/middleware.md'] = { guide: true, title: 'Middleware', acquit: true, markdown: true };
+docs['docs/plugins.md'] = { guide: true, title: 'Plugins', markdown: true };
+docs['docs/subdocs.md'] = { guide: true, docs: true, title: 'SubDocuments', markdown: true };
+docs['docs/documents.md'] = { guide: true, docs: true, title: 'Documents', markdown: true };
+docs['docs/models.md'] = { guide: true, title: 'Models', markdown: true };
+docs['docs/queries.md'] = { guide: true, title: 'Queries', markdown: true };
+docs['docs/populate.md'] = { guide: true, title: 'Query Population', markdown: true };
+docs['docs/migration.md'] = { guide: true, title: 'Migration Guide', markdown: true };
+docs['docs/migrating_to_5.md'] = { guide: true, title: 'Migrating to Mongoose 5', markdown: true };
+docs['docs/migrating_to_6.md'] = { guide: true, title: 'Migrating to Mongoose 6', markdown: true };
+docs['docs/migrating_to_7.md'] = { guide: true, title: 'Migrating to Mongoose 7', markdown: true };
+docs['docs/connections.md'] = { guide: true, title: 'Connecting to MongoDB', markdown: true };
+docs['docs/lambda.md'] = { guide: true, title: 'Using Mongoose With AWS Lambda', markdown: true };
+docs['docs/geojson.md'] = { guide: true, title: 'Using GeoJSON', acquit: true, markdown: true };
+docs['docs/transactions.md'] = { guide: true, title: 'Transactions', acquit: true, markdown: true };
+docs['docs/deprecations.md'] = { guide: true, title: 'Deprecation Warnings', markdown: true };
+docs['docs/further_reading.md'] = { title: 'Further Reading', markdown: true };
+docs['docs/jest.md'] = { title: 'Testing Mongoose with Jest', markdown: true };
+docs['docs/faq.md'] = { guide: true, title: 'FAQ', markdown: true };
+docs['docs/typescript.md'] = { guide: true, title: 'Using TypeScript with Mongoose', markdown: true };
+docs['docs/compatibility.md'] = {
+  title: 'MongoDB Version Compatibility',
+  guide: true,
+  markdown: true
+};
+docs['docs/timestamps.md'] = { title: 'Mongoose Timestamps', markdown: true };
+docs['docs/search.pug'] = { title: 'Search' };
+docs['docs/enterprise.md'] = { title: 'Mongoose for Enterprise', markdown: true };
+docs['docs/sponsors.pug'] = {
+  title: 'Mongoose Sponsors',
+  sponsors
+};
+docs['docs/async-await.md'] = { title: 'Using Async/Await with Mongoose', markdown: true };
+docs['docs/jobs.pug'] = {
+  title: 'Mongoose MongoDB Jobs',
+  jobs
+};
+docs['docs/change-streams.md'] = { title: 'MongoDB Change Streams in NodeJS with Mongoose', markdown: true };
+docs['docs/lodash.md'] = { title: 'Using Mongoose with Lodash', markdown: true };
+docs['docs/incompatible_packages.md'] = { title: 'Known Incompatible npm Packages', markdown: true };
+
+// do sub-directories
+require('./typescript');
```

Comment:
I think you would need to pass `docs` in here somehow? I don't think `require('./typescript')` actually modifies the exports of this file

Reply:
true, fixed with 0681d38f0907e4d5fc8c53c4c8e100f105f50caa